spec/mercure.md

```diff
@@ -27,9 +27,14 @@ organization = "Les-Tilleuls.coop"
 
 .# Abstract
 
-Mercure is a protocol enabling the pushing of data updates to web browsers and other HTTP clients in
+Mercure provides a common publish-subscribe mechanism for public and private web resources.
+Mercure enables the pushing of any web content to web browsers and other clients in
```

Comment:
```suggestion
Mercure enables the pushing of any content to web browsers and other clients in
```

Reply:
I reused the same working as in https://www.w3.org/TR/websub/.